website/docs/r/slo_folder.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: 'sumologic'
+page_title: 'SumoLogic: sumologic_slo_folder'
+description: |-
+  Provides the ability to create, read, delete, and update folders for SLOs.
+---
+
+# sumologic_slo_folder
+
+Provides the ability to create, read, delete, and update folders for SLO's.
+
+## Example SLO Folder
+
+NOTE: SLO folders are considered a different resource from Library content and monitor folders.
+
+```hcl
+resource "sumologic_slo_folder" "tf_slo_folder" {
+  name        = "Terraform Managed SLO's"
+  description = "A folder for SLO's managed by terraform."
+}
+```
+
+## Example Creating SLO Folders and then SLO's in them
+
+```hcl
+
+resource "sumologic_slo_folder" "tf_slo_folder" {
+  name        = "slo-tf-folder"
+  description = "Root folder for SLO created for testing"
+  parent_id   = "0000000000000001"
```

Comment:
should we change the name/description - `Root folder` seems like it refers to the root itself (which should get created by default, and not via terraform config)?